Ska/engarchive/update_archive.py

```diff
@@ -741,7 +741,7 @@ def i_colname(dat):
                     # record.  Thus file0['tstop'] is generally very close to file1['tstart'],
                     # but in this case there can a tiny overlap (<< 1 ms) that triggers an
                     # overlap entry.
-                    logger.verbose('WARNING: Unexpected null file overlap file0=%s file1=%s'
+                    logger.verbose('INFO: Unexpected null file overlap file0=%s file1=%s'
```

Comment:
@javierggt - completely agreed in principle.  In fact the default logger output includes the log level at the beginning e.g. https://docs.python.org/3/howto/logging.html#a-simple-example.  

Alas this PR is just a quick-fix since updating the entire cheta logging to be rational would be a much bigger job.

Reply:
I figured you would say that. So I guess it's "approved"